website/docs/r/ssm_association.html.markdown

```diff
@@ -35,6 +35,8 @@ The following arguments are supported:
 * `parameters` - (Optional) A block of arbitrary string parameters to pass to the SSM document.
 * `schedule_expression` - (Optional) A cron expression when the association will be applied to the target(s).
 * `targets` - (Optional) A block containing the targets of the SSM association. Targets are documented below. AWS currently supports a maximum of 5 targets.
+* `max_concurrency` - (Optional) The maximum number of targets allowed to run the association at the same time.
```

Comment:
I'd recommend adding here and to max errors documentation below that it accepts a number or percentage with percent sign 👍